validator/modules/llm_judge/__init__.py

```
@@ -0,0 +1,267 @@
+        params = {
+            "model": model,
+            "messages": messages,
+            "temperature": self.config.temperature,
+            "seed": random.randint(0, 10000),
+        }
+
+        try:
+            completion = self.client.chat.completions.create(**params)
+            return completion.choices[0].message.content
+        except Exception as e:
+            raise LLMJudgeException(f"API call failed: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)